src/wrappers/any.jl

```diff
@@ -20,21 +20,54 @@ function __init__()
     @initcxx
 end  # __init__()
 
-function node_manager_port()
+function parse_ray_args()
+   #==
+    "Starting agent process with command: ...
+    --node-ip-address=127.0.0.1 --metrics-export-port=60404 --dashboard-agent-port=60493
+    --listen-port=52365 --node-manager-port=58888
+    --object-store-name=/tmp/ray/session_2023-08-14_14-54-36_055139_41385/sockets/plasma_store
+    --raylet-name=/tmp/ray/session_2023-08-14_14-54-36_055139_41385/sockets/raylet
+    --temp-dir=/tmp/ray --session-dir=/tmp/ray/session_2023-08-14_14-54-36_055139_41385
+    --runtime-env-dir=/tmp/ray/session_2023-08-14_14-54-36_055139_41385/runtime_resources
+    --log-dir=/tmp/ray/session_2023-08-14_14-54-36_055139_41385/logs
+    --logging-rotate-bytes=536870912 --logging-rotate-backup-count=5
+    --session-name=session_2023-08-14_14-54-36_055139_41385
+    --gcs-address=127.0.0.1:6379 --minimal --agent-id 470211272
+   ==#
     line = open("/tmp/ray/session_latest/logs/raylet.out") do io
         while !eof(io)
             line = readline(io)
-            if contains(line, "NodeManager server started")
+            if contains(line, "Starting agent process")
```

Comment:
wait, why does this need to change?

Reply:
We're sourcing info from a different line in the logs that contains more info:

before only node manager port was given:
```
47:[2023-08-14 17:26:51,068 I 50732 11107287] (raylet) grpc_server.cc:140: NodeManager server started, listening on port 62065.
```

now we get everything the raylet was launched with:
```
48:[2023-08-14 17:26:51,071 D 50732 11107287] (raylet) agent_manager.cc:87: Starting agent process with command: /Users/glenn/.julia/dev/ray_core_worker_julia_jll/venv/bin/python3 -u /Users/glenn/.julia/dev/ray_core_worker_julia_jll/deps/ray/python/ray/dashboard/agent.py --node-ip-address=127.0.0.1 --metrics-export-port=65329 --dashboard-agent-port=63056 --listen-port=52365 --node-manager-port=62065 --object-store-name=/tmp/ray/session_2023-08-14_17-26-49_606540_50725/sockets/plasma_store --raylet-name=/tmp/ray/session_2023-08-14_17-26-49_606540_50725/sockets/raylet --temp-dir=/tmp/ray --session-dir=/tmp/ray/session_2023-08-14_17-26-49_606540_50725 --runtime-env-dir=/tmp/ray/session_2023-08-14_17-26-49_606540_50725/runtime_resources --log-dir=/tmp/ray/session_2023-08-14_17-26-49_606540_50725/logs --logging-rotate-bytes=536870912 --logging-rotate-backup-count=5 --session-name=session_2023-08-14_17-26-49_606540_50725 --gcs-address=127.0.0.1:6379 --minimal --agent-id 470211272
```

Reply:
ah nice.  looks like that gives the object store and raylet sockets paths too!

Reply:
I added those as parsed vars as well